src/utils/index.ts

```diff
@@ -56,9 +56,7 @@ export function formatCode(
 ) {
   return code.replace(/{{(.*?)}}/g, (match) => {
     const name = match.split(/{{|}}/).filter(Boolean)[0];
-    return JSON.stringify(
-      args && name in args ? args?.[name] : optionalParams?.[name],
-    );
+    return args?.[name] !== undefined ? args?.[name] : optionalParams?.[name];
```

Comment:
so we d'ont stringify it anymore ? would this be a breaking change ?

Reply:
maybe not, since all data json are modified. But how should null be handled ?

Reply:
it's in the code block now, cause this was returning window['undefined'] otherwise if 'l' wasn't set

Reply:
Nit, but wouldn't both a null + undefined check be better here?

```suggestion
    return args[name] ??  optionalParams?.[name];
```

Reply:
i'm not sure what's better... in theory an arg could have a key with a null value on purpose?

Reply:
I guess that a null value is an intentional value set from users 🤔